packages/resolver/sources/fs.js

```diff
@@ -57,6 +57,22 @@ class FS {
     ];
 
     let body, filePath;
+
+    if (importPath === "truffle/Console.sol") {
```

Comment:
this should be its own source, I think

Reply:
@gnidan Not sure what "be its own source" means?

Reply:
resolver defines sources (this one is `resolver/sources/fs.js`) - we can make `resolver/sources/internals.js` or somesuch... should be much cleaner that way

Reply:
@CruzMolina is probably more knowledgeable about this here.

Reply:
I consider this request out of scope for this initial PR.

@gnidan, want to open an issue and tackle it as some later point?

:shipit: